project/issues_due_to_nesting.md

```diff
@@ -0,0 +1 @@
+Nested charts only support enablement by boolean, not by complex tests such as string compares.
```

Comment:
What does this mean? Shouldn't this move into the FAQ, preferably with a more explanatory description, I have absolutely no clue what this statement means and why we need a dedicated MD file for this.

Reply:
Charts.yaml -> dependency -> condition property can only reference a boolean value, not a complex test such as
```
if eq .Values.type "secondary"
```

Reply:
The FAQ is intended for users. This file seems to be intended for contributors.

Reply:
Ok getting the point this is contributors driven. Discussed during the meeting, Find it hard to grasp the intent/meaning of that single sentence. Was hoping it could be improved so it is more clear to any reader what we mean by this single sentence in this isolated file that doesn't provide additional context.

Reply:
We can fix it, but lets do that in a followon, this has been WAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAY too long in review already.

Reply:
Not a fan of fixing unclear things in followup PRs, the whole point of a review is to adress them in the PR, but trusting you on creating that doc improvement PR as followup.

Reply:
https://github.com/spiffe/helm-charts/pull/450 filed as promised.